libraries/machine_name.rb

```diff
@@ -1,28 +1,53 @@
 module MacOS
   module MachineName
-    def scutil
-      '/usr/sbin/scutil'
+    def conform_to_rfc1034(hostname)
```

Comment:
A more descriptive name? Perhaps `validate_domain_name` or `validate_hostname`?

Reply:
I wanted to make it specific to standardization. It's also performing an action on the entered hostname, so I chose a verb rather than a predicate method name.

How about `conform_to_dns_standards`?

Reply:
Works for me 👍